JSDemos/Demos/VectorMap/DynamicViewport/React/App.js

```diff
@@ -53,33 +65,54 @@ class App extends React.Component {
           ref={this.storeMapInstance}
           onZoomFactorChanged={this.zoomFactorChanged}
           onCenterChanged={this.centerChanged}>
+          <ControlBar zoomVisible={this.state.zoomVisible} panVisible={this.state.panVisible} />
           <Layer dataSource={mapsData.world} />
         </VectorMap>
+
         <div className="options">
           <div className="caption">Options</div>
-          <div className="option">
-            <span>Continent</span>&nbsp;
-            <SelectBox
-              dataSource={viewportCoordinates}
-              displayExpr="continent"
-              valueExpr="coordinates"
-              value={this.state.coordinates}
-              onValueChanged={this.continentChanged}
-            />
-          </div>
-          <div className="option">
-            <span>Zoom factor</span>&nbsp;
-            <TextBox
-              value={this.state.zoomFactor}
-              readOnly={true}
-            />
-          </div>
-          <div className="option">
-            <span>Center</span>&nbsp;
-            <TextBox
-              value={this.state.center}
-              readOnly={true}
-            />
+          <div className="wrapper-option">
+
+            <div className="column">
+              <div className="option">
+                <span>Continent</span>&nbsp;
+                <SelectBox
+                  dataSource={viewportCoordinates}
+                  displayExpr="continent"
+                  valueExpr="coordinates"
+                  value={this.state.coordinates}
+                  onValueChanged={this.continentChanged}
+                />
+              </div>
+              <div className="option">
+                <span>Zoom factor</span>&nbsp;
+                <TextBox
+                  value={this.state.zoomFactor}
+                  readOnly={true}
+                />
+              </div>
+              <div className="option">
+                <span>Center</span>&nbsp;
+                <TextBox
+                  value={this.state.center}
+                  readOnly={true}
+                />
+              </div>
+            </div>
+            <div className="column">
+              <div className="option">
+                <span>pan Visible</span>&nbsp;
+                <Switch
+                  value={this.state.panVisible}
+                  onValueChanged={this.panVisibleChanged} />
```

Comment:
In react should uses onValueChange option - https://js.devexpress.com/Documentation/Guide/React_Components/State_Management/?search=onValueChange=